velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -99,6 +99,29 @@ struct TimestampWithTimezoneSupport {
 
 } // namespace
 
+template <typename T>
+struct CurrentTimezoneFunction : public TimestampWithTimezoneSupport<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  std::optional<int64_t> sessionTzID_;
```

Comment:
Doesn't seem like you are using this optional variable. Please remove it.

Reply:
Thanks, removed.